tests/core/test_model.py

```diff
@@ -113,6 +113,39 @@ def test_classificationtask_train(tmpdir: str, metrics: Any):
     assert "test_nll_loss" in result[0]
 
 
+def test_nested_task(tmpdir):
+    model = nn.Sequential(nn.Flatten(), nn.Linear(28 * 28, 10), nn.Softmax())
+    train_dl = torch.utils.data.DataLoader(DummyDataset())
+    val_dl = torch.utils.data.DataLoader(DummyDataset())
+    child_task = ClassificationTask(model, loss_fn=F.nll_loss)
+
+    class Parent(ClassificationTask):
```

Comment:
I think ``__setattr__`` function provides an option to have multiple nested ``Task`` objects? Can we get a test which calls functions from 2 or more children to see if the functions are being called properly?

Reply:
Done :smiley: